hardware/Daikin.cpp

```diff
@@ -125,7 +125,7 @@ bool CDaikin::StopHardware()
 
 void CDaikin::Do_Work()
 {
-	int sec_counter = Daikin_POLL_INTERVAL - 2;
+	sec_counter = Daikin_POLL_INTERVAL - 2; // Trigger immediatly (in 2s) a POLL after startup.
```

Comment:
Could you change this variable so it starts with m_ as all class member variables should do?